### DISCUSSION TITLE
Option for Permanent Baby Mobs via a Bucket of Milk

### DISCUSSION THREAD

**Discussion by ghost** [PurpurMC/Purpur#722] [Feature Requests]
Hey!

There's an option in EssentialsX that allows you to turn a mob into its baby form by hitting it with a bucket of milk. This is only temporary as it'll age as normal and eventually grow up again. I was wondering if there's a way to make it 'permanent' (or at least last a very long time) by altering the age of the mob to a huge negative number when you hit it with the milk

Thanks!

**Comment by TetraTheta** [comment]
I think this suggestion should go to EssentialsX, rather than Purpur.

It is EssentialsX that provides way to convert mob to baby, not Purpur. If Purpur provides way to set permanant baby state, isn't it 'fair' or 'consistent' that Purpur should provide way to convert mob to baby state? But as you know, its EssentialsX's own function. No need to implement thing that already present and well known.

**Reply by ghost** [reply]
I'm happy to suggest it to EssentialsX too but I personally think it's a feature that suits Purpur

**Comment by VonSparq** [comment]
Both [Tweakin](https://www.spigotmc.org/resources/tweakin-1-16.93444/) and [MineTweaks](https://www.spigotmc.org/resources/minetweaks.96757/) have this option, I know it's not as good as integration into Purpur but it's there if you need it.
